python/GafferUI/GLWidget.py

```diff
@@ -216,7 +216,7 @@ def __init__( self, format ) :
 
 		QtWidgets.QGraphicsView.__init__( self )
 
-		self.setObjectName( "gafferGLWidget" )
+		self.setProperty( "gafferGLWidget", True )
```

Comment:
Can this not use `gafferClassName` somehow?

Reply:
It derives from `QtWidgets.QGraphicsView` instead of `GafferUI.Widget` It felt wrong to use `gafferClass` in that case.

Reply:
We just made `styleClassName()` private to `Widget` too 😆

Reply:
I was wondering if we could use a CSS selector that said "QGraphicsView inside a widget where gafferClassName==GLWidget". No worries though, let's leave it.

Reply:
https://github.com/GafferHQ/gaffer/pull/3229/commits/977fb88b635c6d63bfedc947ee83037fb7a9c3c9 - it was 2s to fix it and keeps us consistent :)